src/system/sync/fileCache/__tests__/getCurrentSyncProgress.tests.ts

```diff
@@ -0,0 +1,44 @@
+import { getCurrentSyncProgress } from "system/sync/fileCache/getCurrentSyncProgress"
+import { getFileFromCache } from "system/sync/fileCache/getFileFromCache"
+
+jest.mock("system/sync/fileCache/getFileFromCache")
+
+describe("getCurrentSyncProgress", () => {
+  const mockGetFielFromCache = getFileFromCache as jest.Mock
```

Comment:
```suggestion
  const mockGetFileFromCache = getFileFromCache as jest.Mock
```
Minor typo in the mock